test/unit/dash.models.DashManifestModel.js

```diff
@@ -1508,5 +1508,92 @@ describe('DashManifestModel', function () {
             })
 
         })
+
+        describe('get CMCD data from Manifest', () => {
+
+            it('should return client data reporting from manifest', () => {
+                const manifestData = {
+                    ServiceDescription:[{
+                        'ClientDataReporting':{
+                            'CMCDParameters': {
+
+                            },
```

Comment:
Please also check for the right default values here for `mode` and `includeInRequests`